### PR TITLE
update sponsors list following OASIS audit

### PR DESCRIPTION
**Pull request by jorydotcom**
This PR updates the list of OSLC sponsors which reflects current OP sponsorship & OASIS member participation per OASIS records.

- removes members who have dropped their membership
- adds members and sponsors who have joined to list
- adds logo image as permitted by sponsor

### CODE REVIEW

File: content/about/_index.md [hunk 1]
```diff
@@ -92,8 +91,12 @@ sponsors_img_size = [
 
 sponsors_others_list = [
   "Austrian Institute of Technology",
+  "Axel Reichwein",
   "Bank of America",
+  "Cisco Systems",
+  "Ericsson Fujitsu Limited",
```
Comment (berezovskyi) [inline content/about/_index.md:97]: I think these should be 2 different companies, Ericsson, and Fujitsu Limited.

Comment (berezovskyi) [reply content/about/_index.md:97]: example https://www.ericsson.com/en/press-releases/2018/10/fujitsu-and-ericsson-team-up-on-5g-partnership

Comment (jorydotcom) [reply content/about/_index.md:97]: I will need to consult with Dee & Mary Beth at OASIS. This is how the membership is attributed in the OASIS Member management system.

Comment (jorydotcom) [reply content/about/_index.md]: Thanks again for your patience, @berezovskyi, Mary Beth confirmed your correction & the fix has been pushed.

**Comment by jorydotcom** [comment]
Hi @berezovskyi can you confirm my last push addressed the changes you requested?

**Comment by berezovskyi** [comment]
my apologies @jorydotcom for the delay!

**Pull request merged**